src/MPManager.cpp

```diff
@@ -39,8 +39,8 @@ bool MPManager::initialize()
     if (!AppDaemon::isEmulationMode())
     {
 #if defined(Q_OS_WIN)
-        connect(UsbMonitor_win::Instance(), SIGNAL(usbDeviceAdded()), this, SLOT(usbDeviceAdded()));
-        connect(UsbMonitor_win::Instance(), SIGNAL(usbDeviceRemoved()), this, SLOT(usbDeviceRemoved()));
+        connect(UsbMonitor_win::Instance(), SIGNAL(usbDeviceAdded(QString)), this, SLOT(usbDeviceAdded(QString)));
+        connect(UsbMonitor_win::Instance(), SIGNAL(usbDeviceRemoved(QString)), this, SLOT(usbDeviceRemoved(QString)));
```

Comment:
The signal/slots signature changed from usbDeviceAdded() to usbDeviceAdded(QString), this change was not reflected in UsbMonitor_mac/UsbMonitor_linux, and this will build but fail at runtime. You need to change the signals/slots in those class too.

Reply:
This has been only implemented for Windows yet (because of the crash), Mac/Linux is still using the usbDeviceAdded() without parameter.

Reply:
Yes I know but you changed the signature of the slots in MPManager
https://github.com/mooltipass/moolticute/pull/422/files#diff-afdcd22bf9d85fd4e0aad3e2ee1eb243R58

So it will be changed for windows, but also for linux and macos

Reply:
I kept the signal without the parameter too, and for mac/linux connecting that one: https://github.com/mooltipass/moolticute/blob/21d3bd8d6c174cf269ce499f69fcfaac102b7d77/src/MPManager.cpp#L50
I tested it on Linux and called the correct function, hence I thought qt connect can handle overloads, but I can change the name of the function with QString arg instead of the overload.

Reply:
Ah you are right, I misread, you kept the old slots... It's ok then